.circleci/config.yml

```diff
@@ -27,6 +27,16 @@ apt-run: &apt-install
     sudo apt install -y libopenjp2-7
 
 jobs:
+  twine-check:
```

Comment:
Why are we adding this here?

Reply:
So we know we do not break the sdist? Without it, I would not have known the html table would not render.

Reply:
but why on circle and not GHA?

Reply:
It was easier? I could create a tox env for this and add it to the cron actions?

Reply:
Does our actions config allow arbitrary commands like this?

Reply:
Not all our actions things have to run in tox. You can add a not-openastronomy workflows jobs in the same way: https://github.com/astropy/astropy/blob/main/.github/workflows/ci_workflows.yml#L20

Reply:
I wonder if we should add this to one of the OA publish workflows somewhere :thinking:

Reply:
Should be added to the source dist one no?